### PR TITLE
Fix/123 model deployment node selector

### PR DESCRIPTION
**Pull request by vlad-tokarev**

### CODE REVIEW

**Review by karbyshevds — APPROVED** [review]

File: src/main.go [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+var (
+	log = logf.Log.WithName("node-selector-controller")
+	WebhookV1Path = "/mutate-v1-pod"
+)
+
+
+
+var mainCmd = &cobra.Command{
+	Use:   "node-selector-webhook",
+	Short: "odahu-flow API server",
```
Comment (karbyshevds) [inline src/main.go]: Is it correct desc?

File: version.info [hunk 2]
```diff
@@ -0,0 +1,19 @@
+#
+#    Copyright 2019 EPAM Systems
```
Comment (karbyshevds) [inline version.info]:
```suggestion
#    Copyright 2020 EPAM Systems
```

**Review by karbyshevds — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -1,2 +1,8 @@
...
-a node selector to pods in specific namespace
\ No newline at end of file
+This repository contains webhook for kubernetes that assign 
+a node selector to pods in specific namespace
+
+### How it works
+
+Webhook monitor all pods in namespaces that 
+labeled by "odahu/node-selector-webhook" (label should exists)
+and add configured tolerations and node selector for all pods
```
Comment (maxkochubey) [inline README.md]: Typos fixed.

```suggestion
This repository contains webhook for kubernetes that assigns 
a node selector to pods in specific namespace

### How it works

Webhook monitors all pods in namespaces that 
labeled by "odahu/node-selector-webhook" (label should exist)
and adds configured tolerations and node selector for all pods 
```

File: scripts/Jenkinsfile [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+        stage('Checkout and set build vars') {
+            steps {
+                cleanWs()
+                checkout scm
+                script {
+                    // import ODAHU components
+                    sshagent(["${env.legionCicdGitlabKey}"]) {
+                        print ("Checkout CI/CD repo")
+                        sh"""#!/bin/bash -ex
+                        mkdir -p \$(getent passwd \$(whoami) | cut -d: -f6)/.ssh && ssh-keyscan git.epam.com >> \$(getent passwd \$(whoami) | cut -d: -f6)/.ssh/known_hosts
```
Comment (maxkochubey) [inline scripts/Jenkinsfile]: Let's get rid of hardcoded hostname.

```suggestion
                        export GIT_SSH_COMMAND="ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no"
```

File: src/main.go [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+	}
+
+	// Setup a Manager
+	log.Info("Setting up manager")
+	mgr, err := manager.New(k8s_config.GetConfigOrDie(), manager.Options{
+		Namespace: "model_deployment",
+		Port: appConfig.Port,
+	})
+	if err != nil {
+		log.Error(err, "Unable create manager")
```
Comment (maxkochubey) [inline src/main.go]: Typo fixed.

```suggestion
		log.Error(err, "Unable to create manager")
```

File: scripts/Jenkinsfile (hunk 4 above)
Comment (maxkochubey) [reply scripts/Jenkinsfile:83]: Let's get rid of hardcoded hostname.

**Review by maxkochubey — CHANGES_REQUESTED** [review]
Some suggestions below.

**Review by maxkochubey — APPROVED** [review]

**Review by karbyshevds — APPROVED** [review]

**Pull request merged**